podpac/core/interpolation/test/test_interpolation.py

```diff
@@ -333,3 +334,72 @@ def interpolate(self, udims, source_coordinates, source_data, eval_coordinates,
         outdata = interp.interpolate(srccoords, srcdata, reqcoords, outdata)
 
         assert np.all(outdata == srcdata)
+
+
+class TestHeterogenousInterpolation(object):
+    DATA = np.arange(64).reshape((4, 4, 4))
```

Comment:
Could you also add a case where the data has multiple 'outputs', multi-channel so to speak?

Reply:
Fixed and tested.